index.ts

```diff
@@ -0,0 +1,17 @@
+import { getBodyEnd } from './contributions/epic';
+import {
+  getViewLog,
+  logView,
+  getNumViewsInPreviousDays,
+  getTotalNumViews,
+  clearViewLog
+} from './contributions/viewLog';
+
+export {
+  getBodyEnd,
+  getViewLog,
+  logView,
+  getNumViewsInPreviousDays,
+  getTotalNumViews,
+  clearViewLog
+};
```

Comment:
I _think_ this can be collapsed to the following, if you wanted:

```suggestion
export { getBodyEnd } from './contributions/epic';
export {
  getViewLog,
  logView,
  getNumViewsInPreviousDays,
  getTotalNumViews,
  clearViewLog
} from './contributions/viewLog';
```